components/DynamicComponent.jsx

```diff
@@ -0,0 +1,18 @@
+import dynamic from "next/dynamic";
+
+const components = {
```

Comment:
is it possible to move all components that are custom elements in a folder, and register them automatically? if automatically is not easy, let's have one file in the main repository that takes care of registering things. + it's easy to add others.